herald/src/data/generate-guild-data.js

```diff
@@ -0,0 +1,72 @@
+"use strict"
```

Comment:
This is a tool to generate json data, not a file served by the angular app.  Should move this out of the `herald/src/` directory.  Perhaps put it at `herald/generate-guild-data.js`?